hyperspy/learn/mva.py

```diff
@@ -1334,36 +1598,45 @@ def normalize_poissonian_noise(self, navigation_mask=None,
                 signal_mask = slice(None)
             else:
                 signal_mask = ~signal_mask
-            # Rescale the data to gaussianize the poissonian noise
+
+            # Check non-negative
+            if dc[:, signal_mask][navigation_mask, :].min() <= 0.0:
```

Comment:
I think that ``==0`` is fine. `<=` -> `<`?

Reply:
Done